### DISCUSSION TITLE
no `Option` section in 3DSlicer MONAI Label plugin

### DISCUSSION THREAD

**Discussion by hw-ju** [Project-MONAI/MONAILabel#891] [Q&A] [answered]
Hi there,
In 3DSlicer, even when I turned on `Developer` mode, there's no `Option` section in the left panel:
<img src="https://user-images.githubusercontent.com/90806807/180889522-da88a5b2-bbfe-4364-b746-c373059b6dc6.png" width="400">

<img src="https://user-images.githubusercontent.com/90806807/180889774-573a9874-cebb-41c5-897b-620a25fc663c.png" width="400">

How can I see that `Option` section?

**Comment by SachidanandAlle** [comment, marked as answer]
not this one... pick/check the one under MONAI Label (last but one in the left side list.. above DICOM)

**Reply by diazandr3s** [reply]
Hi @hw-ju,

Please check this video for more information regarding that: https://youtu.be/KjwuFx0pTXU?t=235

**Reply by hw-ju** [reply]
Thanks @SachidanandAlle  @diazandr3s!
Now it works, I should check this one:
<img src="https://user-images.githubusercontent.com/90806807/180893244-1d9fe5c4-6c53-4034-8cc1-1433af2eed28.png" width="400">
